python3/httplib2/__init__.py

```diff
@@ -181,14 +181,15 @@ def safename(filename):
         pass
     if isinstance(filename,str):
         filename=filename.encode('utf-8')
-    filemd5 = _md5(filename).hexdigest().encode('utf-8')
+    filemd5 = _md5(filename).hexdigest()
     filename = re_url_scheme.sub(b"", filename)
-    filename = re_slash.sub(b",", filename)
+    #filename = re_slash.sub(b",", filename)
```

Comment:
Why is it commented out?

Reply:
This line is not necessary anymore if we use `quote`, just wanted a confirmation from someone that it is safe to delete it.

Reply:
Hmm, maybe a more radical and efficient way would be 
```python
re_slash = re.compile(br'\W+')
```
instead of the `quote`